src/main/java/org/jabref/gui/contentselector/ContentSelectorDialogViewModel.java

```diff
@@ -153,8 +154,8 @@ private void addKeywordIfUnique(Field field, String keywordToAdd) {
 
         List<String> existingKeywords = fieldKeywordsMap.getOrDefault(field, new ArrayList<>());
         existingKeywords.add(keywordToAdd);
+        existingKeywords.sort(Comparator.naturalOrder());
         fieldKeywordsMap.put(field, existingKeywords);
-        keywords.add(keywordToAdd);
```

Comment:
Can you please explain why this is removed?

Reply:
On the next line we call `populateKeywords` which clears the `keywords`  and then populates it with keywords from `fieldKeywordsMap`.  Since it makes no difference if we add to `keywords` at this point I simply deleted this line.

Reply:
Ah thanks for the clarification!